app.py

```diff
@@ -1,48 +1,81 @@
 from os import environ
 
 import aws_cdk as cdk
+from aws_cdk import aws_ec2 as ec2
 
 from src.ecs_stack import EcsStack
+from src.helpers.get_package_version import get_alternate_tag_for_edge_package_version
 from src.load_balancer_stack import LoadBalancerStack
 from src.network_stack import NetworkStack
-from src.service_props import ServiceProps
-from src.service_stack import LoadBalancedServiceStack
+from src.service_props import ServiceProps, ServiceSecret
+from src.service_stack import LoadBalancedServiceStack, ServiceStack
+from src.docdb_props import DocdbProps
+from src.docdb_stack import DocdbStack
+from src.bastion_props import BastionProps
+from src.bastion_stack import BastionStack
 
 # get the environment and set environment specific variables
 VALID_ENVIRONMENTS = ["dev", "stage", "prod"]
 environment = environ.get("ENV")
 match environment:
     case "prod":
         environment_variables = {
-            "VPC_CIDR": "10.254.174.0/24",
-            "FQDN": "prod.mydomain.io",
-            "CERTIFICATE_ARN": "arn:aws:acm:us-east-1:XXXXXXXXX:certificate/69b3ba97-b382-4648-8f94-a250b77b4994",
-            "TAGS": {"CostCenter": "NO PROGRAM / 000000"},
+            "VPC_CIDR": "10.253.174.0/24",
+            "FQDN": "prod.modeladexplorer.org",
+            "CERTIFICATE_ID": "dac041fd-e947-4684-a910-fa343adeac33",
+            "TAGS": {"CostCenter": "Model AD-UCI / 123300", "Environment": "prod"},
+            "AUTO_SCALE_CAPACITY": {"min": 2, "max": 4},
+            "EXPLORER_VERSION": "edge",
```

Comment:
IMO, this still seems a little to specific to the app and would require updating on new repo.  How about something a little more generic, like  `DOCKER_PACKAGE_VERSION` or `MONO_REPO_DOCKER_VERSION`?

Also can we get a PR for this same env var parameterization in [agora-infra-v3](https://github.com/Sage-Bionetworks-IT/agora-infra-v3) as well?